administrator/components/com_joomlaupdate/controllers/update.php

```diff
@@ -41,55 +44,77 @@ public function download()
 			// Informational log only
 		}
 
+		$this->input->set('layout', 'download');
+		$this->display();
+	}
+
+	/**
+	 * Step through the download of the update package
+	 *
+	 * @return  void
+	 * @since   __DEPLOY_VERSION__
```

Comment:
```suggestion
	 * @return  void

	 * @since   __DEPLOY_VERSION__
```
Fix code style error reported by drone.